tst/EnergyPlus/unit/AirflowNetworkBalanceManager.unit.cc

```diff
@@ -21174,4 +21174,152 @@ TEST_F(EnergyPlusFixture, AirflowNetwork_TestDefaultBehaviourOfSimulationControl
+        "    6.096000,6.096000,0,  !- X,Y,Z ==> Vertex 2 {m}",
+        "    0,6.096000,0,  !- X,Y,Z ==> Vertex 3 {m}",
+        "    0,6.096000,3.048000;  !- X,Y,Z ==> Vertex 4 {m}",
+        "Construction,",
+        "    PARTITION,             !- Name",
+        "    GYP BOARD;  !- Outside Layer",
+        "Material,",
+        "    GYP BOARD,  !- Name",
+        "    Smooth,                  !- Roughness",
+        "    1.9050000E-02,           !- Thickness {m}",
+        "    0.7264224,               !- Conductivity {W/m-K}",
+        "    1601.846,                !- Density {kg/m3}",
+        "    836.8000,                !- Specific Heat {J/kg-K}",
+        "    0.9000000,               !- Thermal Absorptance",
+        "    0.9200000,               !- Solar Absorptance",
+        "    0.9200000;               !- Visible Absorptance",
+        "AirflowNetwork:SimulationControl,",
+        "   NaturalVentilation, !- Name",
+        "   MultizoneWithoutDistribution, !- AirflowNetwork Control",
+        "   SurfaceAverageCalculation, !- Wind Pressure Coefficient Type",
+        "   , !- Height Selection for Local Wind Pressure Calculation",
+        "   LOWRISE, !- Building Type",
+        "   1000, !- Maximum Number of Iterations{ dimensionless }",
+        "   LinearInitializationMethod, !- Initialization Type",
+        "   0.0001, !- Relative Airflow Convergence Tolerance{ dimensionless }",
+        "   0.0001, !- Absolute Airflow Convergence Tolerance{ kg / s }",
+        "   -0.5, !- Convergence Acceleration Limit{ dimensionless }",
+        "   90, !- Azimuth Angle of Long Axis of Building{ deg }",
+        "   0.36;                    !- Ratio of Building Width Along Short Axis to Width Along Long Axis",
+        "AirflowNetwork:MultiZone:Zone,",
+        "   WEST_ZONE, !- Zone Name",
+        "   Temperature, !- Ventilation Control Mode",
+        "   Temp21, !- Ventilation Control Zone Temperature Setpoint Schedule Name",
+        "   1, !- Minimum Venting Open Factor{ dimensionless }",
+        "   , !- Indoor and Outdoor Temperature Difference Lower Limit For Maximum Venting Open Factor{ deltaC }",
+        "   100, !- Indoor and Outdoor Temperature Difference Upper Limit for Minimum Venting Open Factor{ deltaC }",
+        "   , !- Indoor and Outdoor Enthalpy Difference Lower Limit For Maximum Venting Open Factor{ deltaJ / kg }",
+        "   300000, !- Indoor and Outdoor Enthalpy Difference Upper Limit for Minimum Venting Open Factor{ deltaJ / kg }",
+        "   OnSch, !- Venting Availability Schedule Name",
+        "   Standard;                !- Single Sided Wind Pressure Coefficient Algorithm",
+        "AirflowNetwork:MultiZone:ReferenceCrackConditions,",
+        "   ReferenceCrackConditions, !- Name",
+        "   , !- Reference Temperature{ C }",
+        "   , !- Reference Barometric Pressure{ Pa }",
+        "   ;                   !- Reference Humidity Ratio{ kgWater / kgDryAir }",
```

Comment:
The unit test is failing now that Reference Temperature is required.
I would like to see the unit test expanded to include 2 different AirflowNetwork:MultiZone:ReferenceCrackConditions, one with only the temperature entered, and then another with values (different from the defaults) in each field.
Then, it would be good to trim this unit test down to the smallest amount of input possible. It might work to have only the AirflowNetwork:MultiZone:ReferenceCrackConditions objects and then call only `process_idf` and `getAirflowElementInput`. I'm not 100% certain that will work, but I think it will.